client/src/app/components/add-rule/add-rule.component.ts

```diff
@@ -210,28 +228,28 @@ export class AddRuleComponent implements OnInit {
     this.conditionForm.resetForm();
     this.sourceForm.resetForm();
     this.executionIntervalForm.resetForm();
-    
+
     this.router.navigate(['/list-rules']);
   }
 
   private checkUserSettingsForAgent(params: Array<SourceAgentSettingsParam>) {
```

Comment:
Please add docstrings to all functions as this will otherwise come up as a pre-commit error.